newIDE/app/src/EventsSheet/EventsTree/Instruction.js

```diff
@@ -289,6 +291,31 @@ const Instruction = (props: Props) => {
                 alt="Condition is negated"
               />
             )}
+            {metadata.isAsync() && (
+              <Tooltip
+                title={
+                  <Trans>
+                    This is an asynchronous event.{' '}
+                    <a
+                      href="https://wiki.gdevelop.io/gdevelop5/events/async"
```

Comment:
Change this a tag to use the Link component with `onClick` using `Window.openExternalURL`